src/theme.js

```diff
@@ -15,7 +15,7 @@ export const COLORS = {
   secondaryDisabled: 'rgba(255,255,255,0.2)',
   success: '#71F197',
   text: '#43484d',
-  textFaded: 'rgba(0, 0, 0, 0.6)',
+  textFaded: 'rgba(0, 0, 0, 0.4)',
```

Comment:
checked this in other places, the change doesn't make text worse there.

Reply:
This doesn't really matter but you could do `rgb(162, 162, 162)` instead